scripts/release_scripts/cut_release_or_hotfix_branch.py

```diff
@@ -308,7 +308,10 @@ def execute_branch_cut(target_version, hotfix_number):
     else:
         python_utils.PRINT(
             'Please cherrypick the required PRs and push the branch '
-            'to Github once this script is done.')
+            'to Github once this script is done.\n'
+            'Note: You do not need to push the branch in order to create '
```

Comment:
Note: You do not need to create a branch protection rule for this new branch. The regex for the main release will automatically cover subsequent hotfix branches for that release.

Reply:
Oh ok. Then maybe it makes sense to skip this check for hotfix branches?

Reply:
I don't think that is what the comment implies. This comment is only for hotfix branch which isn't pushed by the script, so there is a confusion when the next step asks user for a branch protection rule. So, the comment clarifies that push isn't required for the rule creation.

Reply:
I think I'm a bit confused. Why is the next step asking the user for a branch protection rule when creating one is not needed? It shouldn't ask in that case, right?

Reply:
The rule is needed:

- Script creates a hotfix branch.
- Script asks user to create a branch protection rule (The rule for main release won't cover the hotfix branch, so a separate rule is needed).
- Script asks the user to cherrypick the commits and then push the branch after script ends. 

The confusion is that user is asked to create a rule but the branch is not yet pushed - so I want to clarify that a push is not required before creation of the rule.

Reply:
Ah, I see, sorry, I misinterpreted! In that case, how about:

`Note: It is fine to push the branch only after creating the branch protection rule and doing all the cherrypicks.`

(The confusion with the original comment is that it says "please push the branch" and then, immediately after, says "You do not need to push the branch".)

Reply:
Got it, done.